starter-code/contact-app/src/app/contact-list/contact-list.component.html

```diff
@@ -1,18 +1,25 @@
-<div>
-  <label for="contact-name"></label>
-  <input type="text" [(ngModel)]="newContact.name">
-</div>
-<div>
-  <label for="contact-email"></label>
-  <input type="text" [(ngModel)]="newContact.email">
-</div>
-<div>
-  <label for="contact-phone"></label>
-  <input type="text" [(ngModel)]="newContact.phoneNumber">
-</div>
-<div>
-  <label for="contact-image"></label>
-  <input type="text" [(ngModel)]="newContact.image">
-</div>
+<p *ngFor ="let contact of contactList">
+  {{contact.name}}
+  {{contact.email}}
+  {{contact.phoneNumber}}
+  <img src=`{{contact.image}}`/>
+</p>
+
+    <div>
+        <label for="contact-name"></label>
+        <input #input type="text" [(ngModel)]="newContact.name">
+      </div>
+      <div>
+        <label for="contact-email"></label>
+        <input #input type="text" [(ngModel)]="newContact.email">
+      </div>
+      <div>
+        <label for="contact-phone"></label>
+        <input #input type="text" [(ngModel)]="newContact.phoneNumber">
+      </div>
+      <div>
+        <label for="contact-image"></label>
```

Comment:
Su input también debería dar contexto al formulario, no se que debo escribir en cual, ejemplo: 
` <label for="contact-name">Nombre: </label>
<input #input type="text" [(ngModel)]="newContact.name">`